.gitignore

```diff
@@ -2,6 +2,7 @@ node_modules
 npm-debug.log
 build
 coverage
+muiReadmeMaster
```

Comment:
Oh my bad. It was this extra thing in my repo that I should get rid of. Thanks for noticing!